### DISCUSSION TITLE
Issue when install the app

### DISCUSSION THREAD

**Discussion by HarminderSinghPHP** [osiset/laravel-shopify#1224] [Q&A]
When install the app show the error (  Osiset\ShopifyApp\Exceptions\ApiException
Unknown error ). Ater that page refresh app is working fine.

Line Number: vendor/osiset/laravel-shopify/src/ShopifyApp/Services/ApiHelper.php:442 

     throw new ApiException(

**Comment by Kyon147** [comment]
Hey @HarminderSinghPHP 

Can you provide more information on how to replicate the issue - I've not seen this before and booted up a new instance of the package yesterday. What version are you using, what PHP version too?

Thanks

**Reply by HarminderSinghPHP** [reply]
When installing the App then shows the error.
screenshot

https://i.imgur.com/ZzSZwgX.png

**Reply by HarminderSinghPHP** [reply]
PHP 7.4 
Laravel 7
